Formula/m/minijinja.rb

```diff
@@ -0,0 +1,21 @@
+class Minijinja < Formula
+  desc "Render Jinja2 templates directly from the command-line to stdout"
+  homepage "https://github.com/mitsuhiko/minijinja"
+  url "https://github.com/mitsuhiko/minijinja/archive/refs/tags/2.3.1.tar.gz"
+  sha256 "c933a1d9f53de26bb776a61b9346e0b202c332b4ff1309db5d0178a9b603ca06"
+  license "Apache-2.0"
+  head "https://github.com/mitsuhiko/minijinja.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    cd "minijinja-cli" do
+      system "cargo", "install", *std_cargo_args
+    end
+    bin.install "target/release/minijinja-cli" => "minijinja"
+  end
+
+  test do
+    assert_match "minijinja-cli #{version}", shell_output("#{bin}/minijinja --version")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Test has been updated.